manageiq-loggers.gemspec

```diff
@@ -1,25 +1,22 @@
-
-lib = File.expand_path("../lib", __FILE__)
-$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
-require "manageiq/loggers/version"
+require "rbconfig"
+require_relative "lib/manageiq/loggers/version"
 
 Gem::Specification.new do |spec|
-  spec.name          = "manageiq-loggers"
-  spec.version       = Manageiq::Loggers::VERSION
-  spec.authors       = ["ManageIQ Authors"]
-
-  spec.summary       = %q{Loggers for ManageIQ projects}
-  spec.homepage      = "https://github.com/ManageIQ/manageiq-loggers"
-  spec.licenses      = ["Apache-2.0"]
+  spec.name     = "manageiq-loggers"
+  spec.version  = Manageiq::Loggers::VERSION
+  spec.authors  = ["ManageIQ Authors"]
+  spec.summary  = %q{Loggers for ManageIQ projects}
+  spec.homepage = "https://github.com/ManageIQ/manageiq-loggers"
+  spec.licenses = ["Apache-2.0"]
 
   # Specify which files should be added to the gem when it is released.
   # The `git ls-files -z` loads the files in the RubyGem that have been added into git.
   spec.files         = Dir.chdir(File.expand_path('..', __FILE__)) do
     `git ls-files -z`.split("\x0").reject { |f| f.match(%r{^(test|spec|features)/}) }
   end
-  spec.bindir        = "exe"
-  spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
-  spec.require_paths = ["lib"]
```

Comment:
I don't think this should be removed as this is how rubygems knows what gem path to put in the load path.  Changing this, I believe, breaks the common case where it's expect that the lib path is on the LOAD_PATH.  To that end, this is why I think the LOAD_PATH mangling exists (to ensure the directory is as expected).

Reply:
So does the gem directory not get put on the LOAD_PATH?

Reply:
Regardless all of this is unrelated to this PR and should be in a separate PR.

Reply:
@Fryguy Ok, I've restored it, and only made the required changes.